content/desktop/use-desktop/builds.md

```diff
@@ -41,33 +26,126 @@ active builders. The list doesn't include builds for inactive builders.
+the builds from the active builds or completed builds section to view detailed
+information, such as logs, a performance breakdown, cache utilization, and
+more.
+
+The build list doesn't include builds for inactive builders. That is, builders
+that you've removed from your system, or builders that have been stopped.
+
 ## Inspect a build
 
 To inspect a build, select the build that you want to view in the list.
+The inspection view contains a number of tabs:
+
+### Info tab
+
+The **Info** tab displays details about the build.
+
+If you're inspecting a multi-platform build, the drop-down menu in the
+top-right of this tab lets you filter the information down to a specific
+platform:
+
+![Platform filter](../images/build-ui-platform-menu.webp?w=400)
+
+The **Source details** section shows the filepath to the
+[frontend](../../build/dockerfile/frontend.md) that was used and, if available,
+information about the source code repository.
+
+The **Build timing** section contains charts showing a breakdown of the build
+execution from various angles.
 
-The **Info** tab displays details about the build job. The details include
-information such as target stage for multi-stage builds, target platforms, and
-version control information, if available.
+- **Real time** refers to the wall-clock time that it took to complete the build.
+- **Accumulated time** shows the total CPU time for all steps.
+- **Cache usage** shows the extent to which build operations were cached.
+- **Parallel execution** shows how much of the build execution time was spent running steps in parallel.
+
+![Build timing charts](../images/build-ui-timing-chart.webp)
+
+The chart colors and legend keys describe the different build operations. Build
+operations are defined as follows:
+
+| Build operation      | Description                                                                     |
+| :------------------- | :------------------------------------------------------------------------------ |
+| Local file transfers | ???                                                                             |
+| File operations      | ???                                                                             |
+| Image pulls          | Time spent pulling images.                                                      |
+| Executions           | Container executions, i.e. defined in a Dockerfile `RUN` instruction.           |
+| HTTP                 | Remote artifact downloads using `ADD`.                                          |
+| Git                  | Same as **HTTP** but for Git URLs.                                              |
+| Result exports       | Time spent exporting the build results.                                         |
+| SBOM                 | Time spent generating the [SBOM attestation](../../build/attestations/sbom.md). |
+| Idle                 | ???                                                                             |
```

Comment:
```suggestion
| Idle                 | ???                                                                             |
```

Idle means this step has been waiting to be solved in BuildKit (queued). This would happen when someone configures max parallelism: https://docs.docker.com/build/ci/github-actions/configure-builder/#max-parallelism

Like you configure max parallelism to 4 but you have 8 solves running so 4 would wait for a free slot if that makes sense.